mqtt/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Use the same base image as prod edgehub images
-ARG base_tag=1.0.6.15-linux-arm32v7
-FROM azureiotedge/azureiotedge-hub-base:${base_tag}
+ARG base_tag=3.1-bionic-arm32v7
+FROM mcr.microsoft.com/dotnet/aspnet:${base_tag}
```

Comment:
Is this right? I thought mqttd got packaged into the edge hub module?

Reply:
This MQTT is solely for the testing purposed. It seemed the `mqtt/docker/linux/arm32v7` was missed in the previous update to remove the base image (and directly use the OS image in the build).